src/plugins/charts/public/services/palettes/helpers.ts

```diff
@@ -30,25 +31,55 @@ function findColorsByStops(
   stops: number[]
 ) {
   const index = stops.findIndex((s) => comparison(value, s) < 0);
-  return colors[index] || colors[0];
+  return colors[index] || colors[colors.length - 1];
 }
 
 function getNormalizedValueByRange(
   value: number,
   { range }: CustomPaletteState,
-  minMax: { min: number; max: number }
+  minMax: { min: number; max: number },
+  isMaxContinuity: boolean,
+  rangeMax: number
```

Comment:
The `rangeMax` is already contained in the `params` argument.

Reply:
thanks, removed from arguments